### PR TITLE
fix: Add `columnNumber` to stack trace

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Adds missing `columnNumber` property to stack trace object
- **Approach**: Add the missing property
- Related: #61 

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**Updates**

- Add property.
- Re-generate code (this affects other files as well)

### Test plan :test_tube:

- Tested using example project

Results:

```json
"stackTrace": [
            {
                "lineNumber": 6971,
                "className": "",
                "fileName": "[http://localhost:37075/main.dart.js"](http://localhost:37075/main.dart.js),
                "methodName": "Object.Cf",
                "columnNumber": 18
            },
            {
                "lineNumber": 57311,
                "className": "",
                "fileName": "[http://localhost:37075/main.dart.js"](http://localhost:37075/main.dart.js),
                "methodName": "a_s.$0",
                "columnNumber": 47
            },
            {
                "lineNumber": 41483,
                "className": "",
                "fileName": "[http://localhost:37075/main.dart.js"](http://localhost:37075/main.dart.js),
                "methodName": "uJ.GD",
                "columnNumber": 15
            },
```

### Author to check :eyeglasses:
- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested 
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

File: example/.gitignore [hunk 1]
```diff
@@ -32,7 +32,6 @@
 /build/
 
 # Web related
-lib/generated_plugin_registrant.dart
```
Comment (sumitramanga) [inline example/.gitignore:35]: Was this on purpose?

**Review by sumitramanga — COMMENTED** [review]
These changes look good! Will need a test on our end before testing. I'll see if @redj4y can look through this up as he has knowledge in source maps

File: example/.gitignore (hunk 1 above)
Comment (miquelbeltran) [reply example/.gitignore:35]: This change was done by the toolchain, I guess it is.

**Review by redJ4y — APPROVED** [review]
The column number is required for us to query source maps, good call. Let's get this in and then we'll look into consuming these properly in the Raygun backend.

**Comment by miquelbeltran** [comment]
Can you approve this @sumitramanga ? seems that @redJ4y review is not enough for GitHub, as he's not on the repo team for some reason

**Review by TheRealAgentK — APPROVED** [review]

**Comment by TheRealAgentK** [comment]
Done @miquelbeltran

**Pull request merged**